Gopkg.lock

```diff
@@ -275,6 +275,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "e7e318025869f4681064b996e4156431b84ab63b135c387631557d0fdde8f77b"
+  inputs-digest = "05426f255be0f6c75739c5c097f90ed18759f1e8da671fcf78d3c891d7e0384c"
```

Comment:
Remember to check if dependencies are up to date, as this warning is being displayed.
```
    default: Warning: the following project(s) have [[constraint]] stanzas in Gopkg.toml:
    default:   ✗  github.com/antonholmquist/jason
    default:   ✗  github.com/spf13/pflag
    default:
    default: However, these projects are not direct dependencies of the current project:
    default: they are not imported in any .go files, nor are they in the 'required' list in
    default: Gopkg.toml. Dep only applies [[constraint]] rules to direct dependencies, so
    default: these rules will have no effect.
    default: Either import/require packages from these projects so that they become direct
    default: dependencies, or convert each [[constraint]] to an [[override]] to enforce rules
```